apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -21,23 +21,27 @@ export default function CurriculumQuickAssign({
   isNewSection,
   updateSection,
   sectionCourse,
+  initialParticipantType,
```

Comment:
Adding this here gives us access to the participant type we need to query the correct course offerings. I call it this so that we can save 'participantType' to represent the truth, whether it is retrieved from the params or is passed in as a prop